### PR TITLE
Construct list of groups for pyramid_fas_openid.

### PR DESCRIPTION
**Pull request by ralphbean**
This uses fedora-infra/pyramid_fas_openid#3.

### CODE REVIEW

File: development.ini [hunk 1]
```diff
@@ -315,7 +315,7 @@ buildroot_overrides.expire_after = 1
 important_groups = proventesters provenpackager releng security_respons packager bodhiadmin
 
 # Groups that can push updates for any package
-admin_packager_groups = provenpackager releng security_respons
+admin_packager_groups = provenpackager releng security_response
```
Comment (lmacken) [inline development.ini]: The `security_response` group currently doesn't exist https://admin.fedoraproject.org/accounts/group/view/security_respons

Comment (ralphbean) [inline development.ini]: Man.  My bad.  Assumptions!

Comment (ralphbean) [inline development.ini]: OK - rebased that out.

**Comment by lmacken** [comment]
:+1: :bridge_at_night:

**Pull request merged**